api/.env.example

```
@@ -43,3 +43,18 @@ AWS_BUCKET_NAME=
 # ==========================================
 FRONTEND_BASE_URL=
 BACKEND_BASE_URL=
+
+# ==========================================
+# AWS S3
+# ==========================================
+AWS_ACCESS_KEY_ID=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The AWS_ACCESS_KEY_ID key is duplicated

(DuplicatedKey)